roles/testnode/tasks/zap_disks.yml

```diff
@@ -32,7 +32,7 @@
   with_items: "{{ ansible_mounts }}"
   when:
     - item.mount != '/' and
-      item.mount != '/var' and
+      not item.mount.startswith('/var') and
```

Comment:
Would `startswith('/var|/tmp|/root|/etc')` work?

Reply:
I don't see why not, should I make that change?

Reply:
Yeah!

Reply:
Looks like I could replace all but the first condition (`!= /`) with a regex. Sound good?